### PR TITLE
Bugfix: Added link to form title hide

### PR DESCRIPTION
**Pull request by simon-estill**
Replaced href="#" with href="../@model.baseURL"

### CODE REVIEW

File: src/Views/Shared/ds-Header.cshtml [hunk 1]
```diff
@@ -10,7 +10,7 @@
             <div class="govuk-header__content">
             @if (displayHeading)
             {
-                <a href="#" class="govuk-header__link govuk-header__link--service-name">
+                <a href="../@Model.BaseURL" class="govuk-header__link govuk-header__link--service-name">
```
Comment (lallysmbc) [inline src/Views/Shared/ds-Header.cshtml:13]: BaseURL is not within SuccessViewModel which breaks success pages for payment and non-payment journeys.

**Comment by simon-estill** [comment]
Changed .../'@Model.BaseUrl to viewData["BaseUrl2"]

File: src/Views/Error/Index.cshtml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ﻿@{
-    ViewData["Title"] = "Error";  
-    Layout = "_ds_ErrorLayout";
+    ViewData["Title"] = "Error";
+    Layout = "_ds_ErrorLayout";
+    ViewData["BaseUrl"] = $"../{Model.BaseURL} ";
```
Comment (lallysmbc) [inline src/Views/Error/Index.cshtml:4]: Error view does not have a ViewModel, form errors due to null binding on Error view, formTitle not required on error view.

File: src/Views/Shared/ds-Header.cshtml [hunk 3]
```diff
@@ -10,7 +11,7 @@
             <div class="govuk-header__content">
             @if (displayHeading)
             {
-                <a href="#" class="govuk-header__link govuk-header__link--service-name">
+                <a href="@baseUrl" class="govuk-header__link govuk-header__link--service-name">
```
Comment (lallysmbc) [inline src/Views/Shared/ds-Header.cshtml:14]: is ViewData required? 

could we use @Model.StartFormUrl, rather then placing it in ViewData

File: src/Views/Shared/ds-Header.cshtml [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ﻿@{
-    var displayHeading = (string)ViewData["Title"] != "Error";
+    var displayHeading = (string)ViewData["Title"] != "Error";
+    var baseUrl = (string)ViewData["BaseUrl"];
```
Comment (lallysmbc) [inline src/Views/Shared/ds-Header.cshtml:3]: is ViewData required?

**Comment by simon-estill** [comment]
Update with feed back

**Comment by simon-estill** [comment]
fixed merge conflicts

File: src/Views/Document/Expired.cshtml [hunk 5]
```diff
@@ -1,6 +1,6 @@
 @{
-    ViewData["Title"] = "Error";
-}
+    ViewData["Title"] = "Error";
+  }
```
Comment (lallysmbc) [inline src/Views/Document/Expired.cshtml:3]: Can you remove this white-space

File: src/Views/Payment/Index.cshtml [hunk 6]
```diff
@@ -0,0 +1,16 @@
+﻿@model form_builder.ViewModels.PaymentViewModel
+@{
```
Comment (lallysmbc) [inline src/Views/Payment/Index.cshtml:2]: This view has been removed, can be removed from the merge.

**Pull request merged**